apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -44,6 +44,14 @@
       "expectedAlertValue": "getMeetingDetails called with shouldGetVerboseDetails: true",
       "expectedTestAppValue": "{\"details\":{\"scheduledStartTime\":\"testStartTime\",\"joinUrl\":\"testJoinUrl\",\"type\":\"oneOnOneCall\",\"originalCaller\":\"testCallerId\",\"dialedEntity\":\"testDnis\",\"trackingId\":\"testTrackingId\"},\"conversation\":{\"id\":\"testConversationId\"},\"organizer\":{\"id\":\"testOrganizerId\",\"tenantId\":\"testTenantId\"}}"
     },
+    {
+      "title": "getMeetingDetailsVerbose API Call - Success",
+      "version": ">2.28.0",
+      "type": "callResponse",
```

Comment:
I am thinking of that for multi host-sdks testing purpose, I will request for adding `hostSdkVersion` as well for new cases.

Reply:
Added, thanks